index.Rmd

```diff
@@ -261,23 +262,18 @@ as(MNDWI, "SpatialPixelsDataFrame") %>%
 
 I loaded required packages
 
-```{r, message=F, warning=F}
-library(threejs)
```

Comment:
Best practice: List all libraries at the beginning of code. See comment in your original libraries calls.